hex_functions.cpp

```diff
@@ -64,4 +15,4 @@ namespace omni
 
     return result;
   }
-} //namespace omni
+}
```

Comment:
eh, this was fine. `namespace omni` still makes me cringe a little, though.

Reply:
It was just a comment at the end of the file. Taking up space...